docs/pages/docs/examples.tsx

```diff
@@ -23,20 +29,221 @@ export default function Docs() {
       </Type>
 
       <Type as="p" look="body18" margin="1.25rem 0 1.5rem 0">
-        Keystone has a wide range of highly configurable features. Our examples are a growing
-        collection of projects you can run locally to learn more about a particular feature of
-        Keystone.
+        A growing collection of projects you can run locally to learn more about Keystone’s
+        capabilities. Each example includes documentation explaining the how and why. Use them as a
+        reference for best practice, and a jumping off point when adding features to your own
+        Keystone project.
+      </Type>
+
+      <GitHubExamplesCTA />
+
+      <Type as="h2" look="heading30" margin="2rem 0 1rem 0" id="base-projects">
+        Base Projects
       </Type>
 
       <Type as="p" look="body18" margin="1.25rem 0 1.5rem 0">
-        Every project comes with its own docs that explain the how and why. They're useful as a
-        reference for best practice, and as a jumping off point when adding features to your own
-        Keystone project.
+        Simple starters to get you up and running with a local Keystone instance and understand the
+        basics of designing{' '}
```

Comment:
Should this be `basics of designing a schema` ? (missing 'a')

Reply:
Fixed in #6528